docs/archive/developer/CucumberSyntax.md

```diff
@@ -37,7 +37,7 @@ foo is greaterThan 5
 
 Operators are converted to English language equivalents for use in cucumber, so 'greaterThan' is expressed as 'greater than'.
 
-* _there is a field `{field}`_, adds a field called `field` to the profile
+* _there is a non nullable field `{field}`_, adds a field called `field` to the profile
```

Comment:
Should I revert changes to this archived documentation?

Reply:
archived stuff should still be accurate, just not important enough to be shown in main docs

archived is probably the wrong name for it